docs/includes/40gen_debops_repos.rst

```diff
@@ -0,0 +1,29 @@
+.. DebOps repositories [[[
+.. This section was generated by: https://github.com/debops/docs/blob/master/docs/docs/bin/gen_rst_defs_using_remote_api
+
+.. _debops.secret: https://github.com/debops/ansible-secret
+.. _debops.apt: https://github.com/debops/ansible-apt
+.. _debops.apt_preferences: https://github.com/debops/ansible-apt_preferences
+.. _debops.auth: https://github.com/debops/ansible-auth
+.. _debops.backporter: https://github.com/debops/ansible-backporter
+.. _debops.boxbackup: https://github.com/debops/ansible-boxbackup
+.. _debops.console: https://github.com/debops/ansible-console
+.. _debops.debops: https://github.com/debops/ansible-debops
+.. _debops.debug: https://github.com/debops/ansible-debug
+.. _debops.dhcpd: https://github.com/debops/ansible-dhcpd
+.. _debops.directories: https://github.com/debops/ansible-directories
+.. _debops.dnsmasq: https://github.com/debops/ansible-dnsmasq
+.. _debops.encfs: https://github.com/debops/ansible-encfs
+.. _debops.etc_services: https://github.com/debops/ansible-etc_services
+.. _debops.etherpad: https://github.com/debops/ansible-etherpad
+.. _debops.ferm: https://github.com/debops/ansible-ferm
+.. _debops.gitlab: https://github.com/debops/ansible-gitlab
+.. _debops.gitlab_ci: https://github.com/debops/ansible-gitlab_ci
+.. _debops.gitlab_ci_runner: https://github.com/debops/ansible-gitlab_ci_runner
+.. _debops.gitusers: https://github.com/debops/ansible-gitusers
+.. _debops.ifupdown: https://github.com/debops/ansible-ifupdown
+.. _debops.java: https://github.com/debops/ansible-java
+.. _debops.kvm: https://github.com/debops/ansible-kvm
+.. _debops.lxc: https://github.com/debops/ansible-lxc
+.. _debops.mailman: https://github.com/debops/ansible-mailman
```

Comment:
The list of repositories is incompliete, what's going on here? Issues with GitHub API?